src/test/scala/ConsumerTestSuite.scala

```diff
@@ -0,0 +1,165 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+import java.util
+import java.util.Properties
+
+import com.bwsw.kafka.reader.Consumer
+import com.bwsw.kafka.reader.entities._
+import org.apache.kafka.clients.consumer._
+import org.apache.kafka.common.{Node, PartitionInfo, TopicPartition}
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.collection.JavaConverters._
+
+class ConsumerTestSuite extends FlatSpec with Matchers {
```

Comment:
http://www.scalatest.org/user_guide/sharing_fixtures#withFixtureOneArgTest
use it to get rid of duplicate steps

Reply:
ok